src/main/java/org/xstefank/webhook/WebHookEndpoint.java

```diff
@@ -51,11 +55,12 @@ private static FormatConfig readConfig() {
         log.info(configFileName);
         File configFile = new File(configFileName);
         ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
-
         try {
-            return mapper.readValue(configFile, FormatConfig.class);
-        } catch (IOException e) {
+            FormatConfig formatConfig = mapper.readValue(configFile, FormatConfig.class);
+            new VerificationHandler(formatConfig).verifyConfiguration();
```

Comment:
looking at this line alone, wouldn't it be better to have a static method on the Verification handler that takes formatConfig as an argument?